go/libraries/doltcore/sqle/database.go

```diff
@@ -94,7 +120,8 @@ func (db *Database) GetTableInsensitive(ctx context.Context, tblName string) (sq
 		return nil, false, err
 	}
 
-	return &DoltTable{name: tblName, table: tbl, sch: sch, db: db}, true, nil
+	db.tables[exactName] = &DoltTable{name: exactName, table: tbl, sch: sch, db: db}
```

Comment:
Probably a micro-optimization, but since maps have bitten me in the past, why not make a new variable, assign to it the new `DoltTable`, set the map value, and return the variable? Instead of retrieving from the map again?

Reply:
Good suggestion, done.